File: ssaExample/ssa_BasicBlock.java
import java.util.Map;

public class ssa_BasicBlock {

	public Integer Index;
	public String Comment;
	public ssa_Function parent;
	public Object[] Instrs;
	public ssa_BasicBlock[] Preds;
	public ssa_BasicBlock[] Succs;
	public ssa_domInfo dom;
	public Integer gaps;
	public Integer rundefers;
}

File: ssaExample/ssa_BasicBlock.kt
class ssa_BasicBlock {

	var Index: Int? = null
	var Comment: String? = null
	var parent: ssa_Function? = null
	var Instrs: Array<Any>? = null
	var Preds: Array<ssa_BasicBlock>? = null
	var Succs: Array<ssa_BasicBlock>? = null
	var dom: ssa_domInfo? = null
	var gaps: Int? = null
	var rundefers: Int? = null
}
